client/src/app/components/client-gridview/client-gridview.component.ts

```diff
@@ -55,4 +56,16 @@ export class ClientGridviewComponent implements OnInit {
       this.changeDetectorRefs.detectChanges();
   }
 
+  updateList(id: number, property: string, event: any): void {
```

Comment:
You can rename this method to updateUser() or something around that. Please also from now on i think its a good idea to add a comment on what the method is doing just on top of it or whatever is the convention so that when others will start implementing it will be obvious what is doing what.